src/vision/calib/MonoCalibrator.hpp

```diff
@@ -0,0 +1,62 @@
+#ifndef MONOCALIBRATOR_HPP
+#define MONOCALIBRATOR_HPP
+
+#include <ros/ros.h>
+#include "CalibrationSettings.hpp"
+#include <ctime>
+#include <iostream>
+#include <opencv2/core.hpp>
+#include <opencv2/core/utility.hpp>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/calib3d.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/videoio.hpp>
+#include <opencv2/highgui.hpp>
+#include <string>
+#include <vector>
+
+using namespace cv;
+using namespace std;
+
+enum class Mode
+{
+    DETECTION = 0,
+    CAPTURING = 1,
+    CALIBRATED = 2
+};
```

Comment:
This is a really generic name for an enum class. Could we namespace-scope this to something like Vision::Mode ?